src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.cc

```diff
@@ -833,4 +709,190 @@ BtreeLBAManager::_update_mapping(
     });
 }
 
+BtreeLBAManager::alloc_extents_ret
+BtreeLBAManager::_alloc_extents(
+    Transaction &t,
+    laddr_hint_t hint,
+    std::vector<alloc_mapping_info_t> &alloc_infos)
+{
+  LOG_PREFIX(BtreeLBAManager::_alloc_extents);
+  ceph_assert(hint.addr != L_ADDR_NULL);
```

Comment:
See the above comment, let's adjust based on the above comment if that's reasonable.